src/quo/components/overlay/view.cljs

```diff
@@ -2,7 +2,26 @@
   (:require
     [quo.components.overlay.style :as style]
     [react-native.blur :as blur]
-    [react-native.core :as rn]))
+    [react-native.core :as rn]
+    [react-native.pure :as rn.pure]))
+
+(defn view-pure
```

Comment:
In other places are just replacing with pure version, why we are keeping both here? Is it WIP?

Reply:
yes, overwise i have to migrate all children for all overlay usage